pkg/depository/create.go

```diff
@@ -129,20 +130,17 @@ func NewCreateDepositoryCmd() *cobra.Command {
 			}
 		},
 	}
-
-	// define flags
+	// Set up command line flags for depository
 	cmd.Flags().StringP("host", "", "", "host URL of depository server")
 	cmd.Flags().StringP("wallet", "w", common.DefaultWalletConfigDir, "wallet path")
 	cmd.Flags().StringP("account", "a", "", "account to be used")
-	cmd.Flags().StringP("name", "n", "", "depository name")
-	cmd.Flags().StringP("contentType", "t", "File", "depository file type")
-	cmd.Flags().StringP("contentID", "i", "", "depository file ID")
-	cmd.Flags().StringP("platform", "p", "bestchains", "depository source platform")
-
-	// bind depository server to flag
-	_ = viper.BindPFlag("saas.depository.server", cmd.Flags().Lookup("host"))
+	// Depository related info
+	cmd.Flags().String("name", "", "depository name")
```

Comment:
For depository info, removed `shorthand`, so user can clearly know these are for `depsitory info` @0xff-dev any comments?

Reply:
`host` removes shorthand because it conflicts with `help`.